packages/mask/src/components/DataSource/usePersonaConnectStatus.ts

```diff
@@ -35,3 +36,20 @@ export function usePersonaConnectStatus() {
         }
     }, [personas, lastRecognized.identifier?.toText(), activatedSocialNetworkUI])
 }
+
+export function useCurrentPersonaConnectStatus() {
+    const personas = usePersonasFromDB()
+    const lastRecognized = useLastRecognizedIdentity()
+    const currentIdentifier = useValueRef(currentPersonaIdentifier)
+    return useMemo(() => {
+        const currentPersona = personas.find((x) => x.identifier.toText() === currentIdentifier)
+        const currentProfile = currentPersona?.linkedProfiles.find((x) => x.identifier === lastRecognized.identifier)
```

Comment:
Sometime, we compare identities by comparing them with the lowered cases.  I think it's a great time for utilities like `isSameIdentity` and `isSamePersona`.